tide-cookies/src/middleware.rs

```diff
@@ -44,7 +44,7 @@ impl<State: Send + Sync + 'static> Middleware<State> for CookiesMiddleware {
 
             let cookie_jar = cookie_data.content.clone();
 
-            cx.extensions_mut().insert(cookie_data);
+            let _ = cx.extensions_mut().insert(cookie_data);
```

Comment:
It would be nice if we have a comment here saying this is a workaround.

Reply:
Done, waiting for the CI.